src/components/CredderRating/CredderRating.js

```diff
@@ -37,7 +37,7 @@ const CredderRating = ({containerStyle = {}, score}) => {
   };
 
   const renderCredderRatingScore = () => {
-    if (!score || score < 0) return 'n/a';
+    if ((!score && score !== 0) || score < 0) return 'n/a';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the above, this change also improves the handling of `score` when it's `0`. The new condition correctly treats `0` as a valid value and not as a falsy one.

```diff
-    if (!score || score < 0) return 'n/a';
+    if ((!score && score !== 0) || score < 0) return 'n/a';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->